README.md

```diff
@@ -37,7 +35,27 @@ npm install
 node server.js
 ```
 
-
 # Contributing
 
-We are very open to new contributors. If you want to contribute to this project, and don't know where to start, look at the [open issues](https://github.com/jrtechs/github-graphs/issues). Once you know what you want to work on, comment on the issue and file a pull request. 
\ No newline at end of file
+We are very open to new contributors. If you want to contribute to this project, and don't know where to start, look at the [open issues](https://github.com/jrtechs/github-graphs/issues). Once you know what you want to work on, comment on the issue and file a pull request.
+
+# API Reference
+
+`https://github-graphs.com/api/friends/<username>`
+
+- `https://api.github.com/users/<username>`
+- `https://api.github.com/users/<username>/followers`
+- `https://api.github.com/users/<username>/following`
+- `https://api.github.com/users/<username>/orgs`
```

Comment:
Doesn't actually get called with the friends request, this is only called when we try to render the users profile

Reply:
I removed the `/orgs` link is in https://github.com/PeterMorganGH/github-graphs/commit/490ff1709dd6f06c4052c6c321d28d96ff426e5f.